public/javascripts/Choropleths/InitializeSubmittedLabels.js

```diff
@@ -105,7 +105,10 @@ function InitializeSubmittedLabels(map, params, adminGSVLabelView, mapData, labe
             pointToLayer: function (feature, latlng) {
                 let style = $.extend(true, {}, geojsonMarkerOptions);
                 style.fillColor = colorMapping[feature.properties.label_type].fillStyle;
-                if (params.includeLabelColor) {
+                if (feature.properties.expired) {
```

Comment:
What happens if `params.includeLabelColor` is `false`, but `feature.properties.expired` is `true`? Then I think you'd be unintentially changing the look of those maps. You should probably just do:
```
if (params.includeLabelColor) {
    if (feature.properties.expired) {
        style.fillColor = 'lightgrey';
        style.color = colorMapping[feature.properties.label_type].missingPanoStrokeStyle;
    } else {
        style.color = colorMapping[feature.properties.label_type].strokeStyle;
    }
}
```